docs/ig.md

```diff
@@ -137,3 +139,21 @@ $ sudo ig list-containers -o json --containername etcd
   }
 ]
 ```
+
+For example, with `--host`, you can get the following output:
+
+```bash
+$ sudo ig trace exec --host
+CONTAINER                                               PID        PPID       COMM             RET ARGS
+
+# Open another terminal.
+$ ls > /dev/null
```

Comment:
I suggest to use a different command in the container and outside to make it easier to understand the `ig` output.

```suggestion
$ cat /dev/null
```